docs/source/daml/intro/1_Token.rst

```diff
@@ -0,0 +1,62 @@
+.. Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+A Token Contract
+================
+
+The first DAML model you will write is a pretty useless one -- a self-issued, non-transferrable token. It represents the minimal possible DAML model and as such serves as an illustration of the most basic concepts:
+
+- Transactions
+- DAML Modules and Files
+- Templates
+- Contracts
+- Signatories
+
+For this section, create a new file called ``Intro_1_Token.daml`` and open it in DAML Studio. If you have installed the DAML SDK, you can start it with ``daml studio`` from a terminal or just open VS Code. Alternatively, use the :download:`supplied source file <daml/Intro_1_Token.daml>`.
+
+DAML Ledger Basics
+------------------
+
+Like most structures called ledgers, a DAML Ledger is a list of *commits*. A *commit* is the final result of the *submission* of a *transaction* by a *party*. The most basic *transactions* are the creation and archival of a *contract*. A contract is *active* on the ledger, if there is a committed transaction that creates it, but there is no commit that archives it again.
```

Comment:
You're defining rather than explaining again. The way you've written implies users should already know what these concepts are, but I think it can be made clearer with a few tweaks.

```suggestion
Like most structures called ledgers, a DAML Ledger is ultimately just a list of *commits*. When we say *commit*, we mean the final result of when a *party* *submits* a *transaction* to that ledger. 

What is a *transaction*? The most basic examples are the creation and archival of a *contract*, which is a concept we'll cover in more detail through this introduction.

After a committed transaction that creates a contract, that contract is *active* on the ledger. But there can then be further commits that *archive* the contract, meaning it's no longer active.
```